singularity_compatibility.rst

```diff
@@ -0,0 +1,149 @@
+.. _singularity_compatibility:
+
+###########################
+ Singularity Compatibility
+###########################
+
+Since the community decided to `move the project into the Linux Foundation
+<https://apptainer.org/news/community-announcement-20211130>`_ with the
+constraint of a name change to the project, it has been a goal of the
+project to minimize the impact to the user base. If you expereince issues making
+the move, please reach out to the `community <https://apptainer.org/help>`_ so
+we can help you!
+
+
+*************************
+ SIF Image Compatibility
+*************************
+
+The {Project} project has decided to make no changes related to the project
+renaming at the image format level. This means that default metadata within
+SIF images and their filesystems will retain the ``singularity`` name without
+change. This will ensure that containers built with {Apptainer} will continue
+to work with installations of Singularity.
+
+This decision was made to ensure that users can continue to package their
+applications and data with {Project} without concerns of image format
+incompatibility when running in a different computing environment or
+collaborating with colleagues that still use a Singularity installation.
+
+
+.. _singularity_environment_variable_compatibility:
+
+***************************************************
+ Singularity Prefixed Environment Variable Support
+***************************************************
+
+{Project} respects environment variables with the ``SINGULARITY_`` and
+``SINGULARITYENV_`` prefixes when their respective ``{ENVPREFIX}_`` and
+``{ENVPREFIX}ENV_`` counterparts are not set.
+
+
+This first command does not use any environment variable compatibility behavior
+as it is using the ``{ENVPREFIX}ENV_`` prefix to command {Project} to create an
+environment variable in the container with the name ``FOO`` and the value
+``BAR``:
+
+.. code::
+
+  $ APPTAINERENV_FOO=BAR apptainer exec docker://alpine env
```

Comment:
For consistency I suggest using `{ENVPREFIX}ENV` and `{command}` here, and likewise in other code blocks below.  It doesn't make a whole lot of difference because this section is clearly apptainer-specific  but I think we may want to do an audit from time to time and look for accidental occurrences where the replacements should be used so I think we might as well use them everywhere.